src/seedu/addressbook/data/person/Person.java

```diff
@@ -25,6 +28,8 @@ public Person(Name name, Phone phone, Email email, Address address, UniqueTagLis
         this.email = email;
         this.address = address;
         this.tags = new UniqueTagList(tags); // protect internal tags from changes in the arg list
+        this.nextSequenceNumber++;
+        this.sequenceNumber = nextSequenceNumber;
```

Comment:
this statement should happen before the previous line, here you are incrementing nextSequenceNumber by 1 before assigning, so that means that your first sequenceNumber is 2 instead.